client/src/hooks/graphql/use-with-record-state.tsx

```diff
@@ -244,17 +244,21 @@ export function useWithRecordState(
     const idx = answers.findIndex((a) => a.answer.question === upload.question);
     if (idx !== -1) {
       const answer = answers[idx];
+      const newTranscript =
+        upload.transcript || upload.transcript === ""
```

Comment:
I think can do upload.transcript !== undefined for the same effect (empty string or string).

Reply:
[af4e67d](https://github.com/mentorpal/mentor-admin/pull/167/commits/af4e67de2ce46a8835a72f9f6d95d0df42c6855b)